src/components/basic-table/components/basic-table-builder/BasicTableBuilder.tsx

```diff
@@ -43,8 +50,38 @@ export const BasicTableBuilder = ({ isEmbeddedTable, children, emptyMessage, ...
     ...padding('0', scale800),
   };
 
+  const [sortColumn, setSortColumn] = useState('');
+  const [sortAsc, setSortAsc] = useState(true);
+
+  const sortedData = useMemo(() => {
+    return data?.slice()?.sort((a: BasicTableRow, b: BasicTableRow) => {
+      const left = sortAsc ? a : b;
+      const right = sortAsc ? b : a;
+      const leftValue = String(left[sortColumn]);
+      const rightValue = String(right[sortColumn]);
+
+      return leftValue?.localeCompare(rightValue, 'en', {
+        numeric: true,
+        sensitivity: 'base',
+      });
+    });
+  }, [sortColumn, sortAsc, data]);
+
+  function handleSort(id: string) {
```

Comment:
Maybe we also want to add the ability to turn off sorting if you click for the third time. We also use this behavior when on aggrid. But something to discuss with UX